client/source/components/questionsAnswers/DisplayPhotos.jsx

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
 import { PhotoContainer } from './styles/Containers.styled';
-import qaData from './exampleDataQA';
+import qaData2 from './exampleDataQA2';
 
 const DisplayPhotos = function CreateDisplayPhotosComponent() {
   return (
     <PhotoContainer>
       <img
-        src={qaData.results[0].answers[3073848].photos[0]}
+        src={qaData2.results[0].answers[3073848].photos[0]}
```

Comment:
Looks like this is using the hardcoded data in qaData2 for now. Once this is updated with data from the API, a suggestion to make this dynamic for each image.

Reply:
Yeah, this feature I haven't touched since the start. Thanks for the reminder though. I'll have a clearer understanding of what to do with the photos feature once I have implemented the Answer form.